### PR TITLE
UI overhaul

### PR DESCRIPTION
**Pull request by ConnorShore**

### CODE REVIEW

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/LoginActivity.java [hunk 1]
```diff
@@ -52,7 +57,7 @@ private void startSignInActivity(){
         startActivityForResult(
                 AuthUI.getInstance()
                         .createSignInIntentBuilder()
-                        .setTheme(R.style.LoginTheme)
+//                        .setTheme(R.style.LoginTheme)
```
Comment (ConnorShore) [inline WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/LoginActivity.java]: Remove commented out code

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+public class SplashScreenActivity extends AppCompatActivity {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Thread background = new Thread() {
+            public void run()
+            {
+                try {
+                    sleep(1500);
```
Comment (ConnorShore) [inline WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java]: Move to variable & config

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                        startActivityForResult(mainActivity, RequestCodes.RC_MAIN_ACTIVITY);
+                    } else {
+                        Intent loginActivity = new Intent(SplashScreenActivity.this, LoginActivity.class);
+                        startActivityForResult(loginActivity, RequestCodes.RC_LOGIN_ACTIVITY);
+                    }
+                    finish();
+                }
+                catch (Exception ex)
+                {
+
```
Comment (ConnorShore) [inline WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java]: Add Log statement in exception

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/main/adapters/RoomAdapter.java [hunk 4]
```diff
@@ -71,20 +88,25 @@ public void onBindViewHolder(@NonNull Viewholder holder, int position)
...
                     });
 
                     return null;
                 });
             }
             else
             {
-                roomManager.turnOffRoomLights(room, (var, status) -> {
+                toggleBackgroundGlow(false, holder.glowCard, Color.TRANSPARENT);
+                RoomManager.getInstance().turnOffRoomLights(room, (var, status) -> {
```
Comment (ConnorShore) [inline WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/main/adapters/RoomAdapter.java:109]: use roomManager variable

File: WattsApp/app/src/main/res/drawable/splash_background.xml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@drawable/bg_gradient"/>
+<!--    <item>-->
```
Comment (ConnorShore) [inline WattsApp/app/src/main/res/drawable/splash_background.xml:5]: remove commented out code

File: WattsApp/app/src/main/res/layout/card_layout.xml [hunk 6]
```diff
@@ -1,70 +1,57 @@
...
+                    android:gravity="center|top">
+
+                    <TextView
+                        android:id="@+id/roomName"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="10dp"
+                        android:layout_marginTop="10dp"
+                        android:gravity="left"
+                        android:text="TEST"
```
Comment (ConnorShore) [inline WattsApp/app/src/main/res/layout/card_layout.xml:33]: Move to style/config (text stuff)

File: WattsApp/app/src/main/res/layout/light_card_layout.xml [hunk 7]
```diff
@@ -1,37 +1,43 @@
...
+                android:orientation="vertical"
+                android:padding="8dp">
+
+                <TextView
+                    android:id="@+id/lightName"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    android:layout_gravity="center_horizontal"
+                    android:layout_weight="1"
+                    android:textSize="@dimen/tile_text_size"
```
Comment (ConnorShore) [inline WattsApp/app/src/main/res/layout/light_card_layout.xml:25]: Move text stuff to style

File: WattsApp/app/src/main/res/layout/light_list_layout.xml [hunk 8]
```diff
@@ -2,15 +2,17 @@
 <com.google.android.material.card.MaterialCardView
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
+    style="@style/Widget.Material3.CardView.Elevated"
```
Comment (ConnorShore) [inline WattsApp/app/src/main/res/layout/light_list_layout.xml]: Move to our own style

File: WattsApp/app/src/main/res/values/app_icon_launcher_background.xml [hunk 9]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="app_icon_launcher_background">#E56611</color>
```
Comment (ConnorShore) [inline WattsApp/app/src/main/res/values/app_icon_launcher_background.xml]: Move color to colors.xml

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/LoginActivity.java (hunk 1 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/LoginActivity.java]: I got

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java (hunk 2 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java]: I got

File: WattsApp/app/src/main/res/layout/light_card_layout.xml (hunk 7 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/layout/light_card_layout.xml:25]: I got

File: WattsApp/app/src/main/res/layout/card_layout.xml (hunk 6 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/layout/card_layout.xml:33]: I got

File: WattsApp/app/src/main/res/layout/light_list_layout.xml (hunk 8 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/layout/light_list_layout.xml]: I got

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java (hunk 3 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/SplashScreenActivity.java]: Got it

File: WattsApp/app/src/main/res/layout/card_layout.xml (hunk 6 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/layout/card_layout.xml:33]: saving for later

File: WattsApp/app/src/main/res/layout/light_card_layout.xml (hunk 7 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/layout/light_card_layout.xml:25]: saving for later

File: WattsApp/app/src/main/res/drawable/splash_background.xml (hunk 5 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/drawable/splash_background.xml:5]: todo saving for later

File: WattsApp/app/src/main/res/layout/light_list_layout.xml (hunk 8 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/layout/light_list_layout.xml]: Done

File: WattsApp/app/src/main/res/values/app_icon_launcher_background.xml (hunk 9 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/res/values/app_icon_launcher_background.xml]: Done

File: WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/main/adapters/RoomAdapter.java (hunk 4 above)
Comment (ConnorShore) [reply WattsApp/app/src/main/java/com/dabloons/wattsapp/ui/main/adapters/RoomAdapter.java:109]: I got

**Review by zaidabuhijleh — APPROVED** [review]

**Pull request merged**